### PR TITLE
Adds functions to facilitate printing of iid and group tables.

### PR DESCRIPTION
**Pull request by elie-elkhoury**
- Helper to print uint64_t
- Functions to print reduced version of group recipient and publisher tables

### CODE REVIEW

File: api/oc_helpers.c [hunk 1]
```diff
@@ -290,6 +290,37 @@ oc_join_string_array(oc_string_array_t *ocstringarray, oc_string_t *ocstring)
   strcpy((char *)oc_string(*ocstring) + len, "");
 }
 
+int
+oc_print_uint64_t(uint64_t number)
+{
+  if (number == 0) {
+    printf("0\n");
```
Comment (tiniuclx) [inline api/oc_helpers.c:297]: Can we use the PRINT macro here instead?

File: api/oc_helpers.c [hunk 2]
```diff
@@ -290,6 +290,37 @@ oc_join_string_array(oc_string_array_t *ocstringarray, oc_string_t *ocstring)
...
+  }
+
+  // Convert the number to a string
+  for (int i = numDigits - 1; i >= 0; i--) {
+    str[i] = '0' + (number % 10);
+    number /= 10;
+  }
+  str[numDigits] = '\0';
+
+  printf("%s\n", str);
```
Comment (tiniuclx) [inline api/oc_helpers.c:319]: Can we not print a \n in here, in favour of choosing to add one at the application level?

File: api/oc_helpers.c (hunk 1 above)
Comment (tiniuclx) [reply api/oc_helpers.c:297]: Discussed in office, a PRINT version of this function will be implemented when needed

**Review by tiniuclx — CHANGES_REQUESTED** [review]

**Review by tiniuclx — APPROVED** [review]

**Review by WAvdBeek — APPROVED** [review]

**Comment by WAvdBeek** [comment]
~format

**Pull request merged**